README.md

```diff
@@ -101,6 +101,7 @@ Organizations that have contributed code to Tock include:
   * Google
   * Helium
   * Atihita
+  * MIT and MIT Lincoln Laboratory
```

Comment:
```suggestion
  * MIT
  * MIT Lincoln Laboratory
```

Reply:
I think these are different orgs, so should be listed differently, even though the contributors might be the same :)